src/StoreKeeper/WooCommerce/B2C/Options/StoreKeeperOptions.php

```diff
@@ -168,6 +174,11 @@ public static function isPaymentSyncEnabled(): bool
         return in_array(self::getSyncMode(), self::MODES_WITH_PAYMENTS, true);
     }
 
+    public static function isShippingMethodSyncEnabled(): bool
```

Comment:
```suggestion
    public static function isShippingMethodAllowedForCurrentSyncMode(): bool
```

The name is little confusing for me, thought it's checking the `SHIPPING_METHOD_USED`

Reply:
Changes implemented.